### PR TITLE
Better wording for configurations. Title search fixed for config page.

### PR DESCRIPTION
**Pull request by melihkorkmaz**

### CODE REVIEW

File: plugins/plugins/frontend/public/localization/plugins.properties [hunk 1]
```diff
@@ -82,25 +82,25 @@ configs.logs.warn = Warning Level
 configs.logs.error = Error Level
 configs.logs.default = Default Level
 
-configs.help.frontend-production = If enabled, serves minified, combined files
+configs.help.frontend-production = If enabled, serves minified, combined files. (what is the cause of disabling or enabling this? Any potential harms?)
```
Comment (ar2rsawseen) [inline plugins/plugins/frontend/public/localization/plugins.properties]: I am sure, this is a comment, rather than a change :)

Comment (gorkem-cetin) [reply plugins/plugins/frontend/public/localization/plugins.properties]: Right, that was my comment for you in fact.. There is a need for a better explanation if you think it helps?

Comment (melihkorkmaz) [reply plugins/plugins/frontend/public/localization/plugins.properties]: :) Sorry for that. I'll be more carefull. I just copied it :(

Comment (ar2rsawseen) [reply plugins/plugins/frontend/public/localization/plugins.properties]: Initial load of dashboard should be faster, due to smaller files and smaller file amount, but when developing a plugin, you need to regenerate them to see changes

Comment (ar2rsawseen) [reply plugins/plugins/frontend/public/localization/plugins.properties]: No probs, thats what reviews are for :)

**Pull request merged**